packages/block-editor/src/components/media-replace-flow/README.md

```diff
@@ -56,6 +56,13 @@ Callback used when media is replaced with an URL. It is called with one argument
 -   Type: `func`
 -   Required: Yes
 
+### onError
+
+Callback called when an upload error happens and receives an error message as an argument.
+
+-   Type: `func`
+-   Required: Yes
```

Comment:
Required seems to be no, now that we left the default handler, right?

Reply:
Good catch.